### DISCUSSION TITLE
Office 365 Add-Ins

### DISCUSSION THREAD

**Discussion by dazu89** [simeononsecurity/Windows-Optimize-Harden-Debloat#55] [Q&A] [answered]
Hello there, I hope you are doing fine!
Two issues related to Microsoft Office 365 (Version 2202 Build 16.0.14931) 64-bit I have been unable to resolve, and would appreciate any advice:
- Microsoft Access Wizards (e.g. Look-up Wizard), Builders and Add-ins are note working, similar to the symptoms described [here](https://learn.microsoft.com/en-us/microsoft-365/troubleshoot/group-policy/office-add-in-not-loaded#symptoms). 
EDIT: On a standard installation, there is a trusted user location under \Root\Office16\ACCWIZ\ which was not present on my installation, adding the folder via policy did not have any effect. However, by going to the group policy User Configuration > Admin Templates > Microsoft Office 2016 > Security Settings > Trust Center  and setting `Allow mix of policy and user locations` to enabled, the folder showed up in Access after an application restart and the Wizards work as expected.
- Microsoft Excel refuses to load the cherished [excel2latex add-in](https://github.com/ivankokan/Excel2LaTeX) with the somewhat obscure prompt "[...] is not a valid add-in."

I scoured the Local Group Policies but could not find the right buttons.

**Comment by simeononsecurity** [comment, marked as answer]
@dazu89 

Macros and custom extensions are often not allowed in hardened environments. What you're talking about is definitely an enterprise feature set that isn't supported by this repo. 
These are all the settings that might affect this issue:
https://www.stigviewer.com/stig/microsoft_office_365_proplus/
https://www.stigviewer.com/stig/microsoft_office_system_2016/
https://www.stigviewer.com/stig/microsoft_excel_2016/

**Comment by 13xd** [comment]


**Reply by simeononsecurity** [reply]
> 您好！我是刘卓彬。我已经收到了您的邮件，我会尽快查看并回复！谢谢！

Unfortunately, this is an English speaking only forum. Please use English moving forward or we will remove the posting.

不幸的是，这是一个只说英语的论坛。 请继续使用英语，否则我们将删除该帖子。
